embassy-stm32/src/spi/mod.rs

```diff
@@ -195,15 +195,20 @@ impl<'d, T: Instance, Tx, Rx> Spi<'d, T, Tx, Rx> {
     }
 
     /// Useful for on chip peripherals like SUBGHZ which are hardwired.
-    /// The bus can optionally be exposed externally with `Spi::new()` still.
     #[allow(dead_code)]
     pub fn new_subghz(
         peri: impl Peripheral<P = T> + 'd,
         txdma: impl Peripheral<P = Tx> + 'd,
         rxdma: impl Peripheral<P = Rx> + 'd,
-        freq: Hertz,
-        config: Config,
+        pclk3_freq: u32,
```

Comment:
You can obtain pclk3_freq with `RccPeripheral` from inside `embassy-stm32`, there should be no need to make the user pass it in.

Reply:
I tried that, but then I would have to include a feature to prevent an error on - for example - stm32l0 build.

Reply:
Correct; I was worried about ramifications in other areas if I tried to push that through on this PR.  I'll work on another PR to address this.